config/config_handler.go

```diff
@@ -76,9 +86,11 @@ type pdConfHandler struct {
 	exit       chan struct{}
 	pdConfCli  pd.ConfigClient
 	reloadFunc func(oldConf, newConf *Config)
+
+	overwriteFunc OverwriteFunc
```

Comment:
Why add the field? I find the function doesn't be called except `InitializeConfig`. 
Or it should be called in `(ch *pdConfHandler) run()` ?